content/yaml/distribution.md

```diff
@@ -16,7 +16,9 @@ All generated artifacts can be published to external services. The available int
 Codemagic uses the [keychain](https://github.com/codemagic-ci-cd/cli-tools/blob/master/docs/keychain/README.md#keychain) utility to manage macOS keychains and certificates.
 {{</notebox>}}
 
-In order to use **automatic code signing** and have Codemagic manage signing certificates and provisioning profiles on your behalf, you need to configure API access to App Store Connect and define the environment variables listed below. Make sure to [encrypt](#encrypting-sensitive-data) the values of the variables before adding them to the configuration file.
+### Setting up automatic code signing
+
+In order to use **automatic code signing** and have Codemagic manage signing certificates and provisioning profiles on your behalf, you need to configure API access to App Store Connect and define the environment variables listed below. Make sure to [encrypt](#encrypting-sensitive-data) the values of the variables before adding them to the configuration file. Note that when using Codemagic UI to encrypt files, they will also be base64 encoded and you will have to base64 decode them in order for the configuration to work. When adding the variables through the UI and exporting the .yaml file, they will however automatically be decoded.
```

Comment:
I recommend formatting the part about encrypting sensitive values as a separate note.
I would also edit the note about encryption as follows:
"Make sure to [encrypt](#encrypting-sensitive-data) the values of the variables before adding them to the configuration file. To properly encrypt the App Store Connect private key and the certificate private key, copy the contents of the file and encrypt it in the Codemagic encryption interface. You can then add the encrypted values to the respective environment variables. If you upload the file for encryption, it will be also base64-encoded and would have to be decoded during the build."